app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt

```diff
@@ -114,30 +118,43 @@ private fun GroupConversationDetailsContent(
     val elevationState by remember { derivedStateOf { lazyListStates[currentTabState].topBarElevation(maxAppBarElevation) } }
 
     val sheetState = rememberModalBottomSheetState(ModalBottomSheetValue.Hidden)
-    val openBottomSheet: () -> Unit = remember { { scope.launch { sheetState.show() } } }
-    val closeBottomSheet: () -> Unit = remember { { scope.launch { sheetState.hide() } } }
+    val openBottomSheet: () -> Unit = remember { { scope.launch { sheetState.animateTo(ModalBottomSheetValue.Expanded) } } }
 
     val deleteGroupDialogState = rememberVisibilityState<GroupDialogState>()
     val leaveGroupDialogState = rememberVisibilityState<GroupDialogState>()
+    val clearConversationContentDialogState = rememberVisibilityState<DialogState>()
 
     val snackbarHostState = remember { SnackbarHostState() }
+
     LaunchedEffect(Unit) {
         messages.collect { snackbarHostState.showSnackbar(it.asString(context.resources)) }
     }
 
-    if (!isLoading) {
-        deleteGroupDialogState.dismiss()
-        leaveGroupDialogState.dismiss()
-    }
```

Comment:
@Garzas  any thoughts ? delete this one but have a LaunchedEffect inside the dialogs that does it for us and it works well